util/cmake/SeqAnBuildSystem.cmake

```diff
@@ -358,6 +358,27 @@ macro (seqan_build_system_init)
         find_package(BZip2)
         find_package(Boost)
         find_package(SeqAn CONFIG REQUIRED)
+        find_package(LibXml2)
+    endif ()
+
+    option(CTD_TEST_ENABLED "Checks if correct ctd export of a tool can be tested automatically." OFF)
+
+    if (LIBXML2_FOUND)
+        find_program (XMLLINT_EXECUTABLE xmllint DOC "The xmllint tool available in the xmllib package.")
+        if (XMLLINT_EXECUTABLE-NOTFOUND)
+            message (STATUS "Could not find xmllint. Not configuring ctd tests.")
+        else ()
+            set(CTD_SCHEMA_ROOT "${PROJECT_BINARY_DIR}/vendor/ctd_schema")
+            execute_process(COMMAND git clone https://github.com/WorkflowConversion/CTDSchema.git ${CTD_SCHEMA_ROOT}
```

Comment:
maybe use `ExternalProject_Add(` instead?

Reply:
Well yes, but it actually wants to build then the source. I didn't try but it did not seem to be the right choice here. I couldn't find an explicit documentation about how to disable the build step.